src/NuGet.Core/NuGet.Configuration/Proxy/ProxyCache.cs

```diff
@@ -99,6 +99,14 @@ public WebProxy GetUserConfiguredProxy()
                 {
                     webProxy.Credentials = new NetworkCredential(userName, password);
                 }
+                
+                var noProxy = _settings.GetValue(SettingsUtility.ConfigSection, ConfigurationContants.NoProxy);
```

Comment:
I'm asking how did you test the usage pattern for this no_proxy in the final product